### ISSUE TITLE
Semeval2018-El-reg

### ISSUE THREAD

**Issue by GiannisPikoulis** [slp-ntua/slp-labs#93]
Καλησπέρα σας. Το πρόβλημα που ορίζεται στο εν λόγω task είναι, δεδομένου του συναισθήματος (anger, fear, joy, sadness) να γίνει regression πάνω στο intensity (arousal) του συναισθήματος? Γιατί βλέπω πως τα αρχικά test data που είχαν δωθεί από το διαγωνισμό ήταν χωρισμένα σε αρχεία της μορφής  "2018-El-reg-En-{EMOTION}-test.txt" και μέσα οι εγγραφές περιείχαν το συναίσθημα και έλειπε το intensity. Οπότε να υποθέσω ότι στα πλαίσια της εργαστηριακής άσκησης πρέπει να κάνουμε finetune 4 regression μοντέλα (pre-trained στην Semeval2017A), ένα για κάθε συναίσθημα και να κάνουμε aggregate τα Pearson CC και για τα 4 συναισθήματα κατά το evaluation? Ευχαριστώ εκ των προτέρων.

**Comment by pxaris** [comment]
Καλημέρα, 
Για το training set θα χρησιμοποιήσετε τα "**train**" δεδομένα και για το test set τα "**dev**". Αγονείστε τελείως τα "test-gold" δεδομένα (τα οποία, όπως αναφέρθηκε, δεν έχουν τιμή για την ένταση του εκάστοτε συναισθήματος).
